src/gui/screen_menu_preheat.cpp

```diff
@@ -14,9 +14,14 @@ class MI_Filament : public WI_LABEL_t {
     virtual void click(Iwindow_menu_t & /*window_menu*/) override {
         const filament_t filament = filaments[T];
         marlin_gcode("M86 S1800"); // enable safety timer
-        marlin_gcode_printf("M104 S%d", (int)filament.nozzle);
+        /// don't use preheat temp for cooldown
+        if (PREHEAT_TEMP >= filament.nozzle) {
```

Comment:
use `if (PREHEAT_TEMP > filament.nozzle)` parameters S and D makes no sense when they are equal

Reply:
In case of equality "M104 S%d" is called.

Reply:
S makes sense in both cases.